README.md

```diff
@@ -41,21 +46,22 @@ One of the goals of this library is to make sure that the it is easy to use.  Ev
 **Velaptor** has the ability to load texture atlas image and JSON data for texture atlas type rendering as well as single image content and sound content.  The content loading system is flexible and has an API that has the ability to be extended to load custom content for your media applications and games.
 
 ### Content Caching
```

Comment:
To set this feature apart from the text, you might want to bold it or make it bigger.
```suggestion
### Content Caching
```